### PR TITLE
Update docker image registries and gradle plugins

### PR DESCRIPTION
**Pull request by driesdeproost**

### CODE REVIEW

File: start.xenit.eu/src/main/java/eu/xenit/alfred/initializr/start/build/root/gradle/GradleProjectGenerationConfiguration.java [hunk 1]
```diff
@@ -65,7 +65,7 @@ public RootGradleBuildContributor gradleBuildProjectContributor(CustomGradleBuil
 
     @Bean
     public GradleWrapperContributor gradle4WrapperContributor() {
-        return new GradleWrapperContributor("4");
+        return new GradleWrapperContributor("6");
```
Comment (tgeens) [inline start.xenit.eu/src/main/java/eu/xenit/alfred/initializr/start/build/root/gradle/GradleProjectGenerationConfiguration.java:68]: but the method is still called `gradle4WrapperContributor` ?

File: start.xenit.eu/src/main/java/eu/xenit/alfred/initializr/start/sdk/alfred/compose/config/ComposeUpGradleTaskConfiguration.java [hunk 2]
```diff
@@ -25,15 +25,7 @@ public ComposeUpGradleTaskConfiguration(DockerImageEnvNameProvider variableNameP
     @Override
     public void accept(GradleTask.Builder builder) {
         this.usesDockerImageFrom.forEach(dockerImageProject -> {
-            String dockerImageEnvName = this.variableNameProvider.get(dockerImageProject);
-
             builder.invoke("dependsOn", "':"+dockerImageProject.getId()+":buildDockerImage'");
-            builder.nested("doFirst", doFirst -> {
```
Comment (tgeens) [inline start.xenit.eu/src/main/java/eu/xenit/alfred/initializr/start/sdk/alfred/compose/config/ComposeUpGradleTaskConfiguration.java:31]: this only works with newer versions of the `eu.xenit.docker` but I don't think it's updated yet ?

Comment (tgeens) [reply start.xenit.eu/src/main/java/eu/xenit/alfred/initializr/start/sdk/alfred/compose/config/ComposeUpGradleTaskConfiguration.java:31]: ah, it is - never mind

File: start.xenit.eu/src/main/java/eu/xenit/alfred/initializr/start/build/root/gradle/GradleProjectGenerationConfiguration.java (hunk 1 above)
Comment (tgeens) [reply start.xenit.eu/src/main/java/eu/xenit/alfred/initializr/start/build/root/gradle/GradleProjectGenerationConfiguration.java:68]: if you want to upgrade the gradle version that is produced, put that in a different PR ?

Comment (driesdeproost) [reply start.xenit.eu/src/main/java/eu/xenit/alfred/initializr/start/build/root/gradle/GradleProjectGenerationConfiguration.java:68]: I opened a separate PR
I will rebase this PR onto the other one once merged

Comment (driesdeproost) [reply start.xenit.eu/src/main/java/eu/xenit/alfred/initializr/start/build/root/gradle/GradleProjectGenerationConfiguration.java]: rebased onto master

**Review by tgeens — APPROVED** [review]

**Pull request merged**